### PR TITLE
AR : Fix the style / structure of download section ..

### PR DESCRIPTION
**Pull request by 3imed-jaberi**
My try to make the files in the download section similar in structure to the original file ..

### CODE REVIEW

**Comment by ghost** [comment]
@nodejs/nodejs-ar , Please have a review.

**Comment by ram-you** [comment]
This sounds good.

File: locale/ar/download/current.md [hunk 1]
```diff
@@ -22,8 +22,7 @@ downloads:
 additional:
     headline: منصات إضافية
     intro: >
-        يقوم أعضاء من مجتمع النود جي اس بالاشراف على نسخ غير رسمية من النود جي اس لمنصات أخرى.
-        يجب التذكير أن هذه النسخ غير مدعومة من الفريق الأساسي للنود جي اس و قد لا تكون على نفس مستوى التطوير الخاص بالاصدارات الحالية للنود جي اس.
+        يقوم أعضاء من مجتمع النود جي اس بالاشراف على نسخ غير رسمية من النود جي اس لمنصات أخرى. يجب التذكير أن هذه النسخ غير مدعومة من الفريق الأساسي للنود جي اس و قد لا تكون على نفس مستوى التطوير الخاص بالاصدارات الحالية للنود جي اس.
```
Comment (Aissaoui-Ahmed) [inline locale/ar/download/current.md]: النود جي اس > Node.js

Comment (3imed-jaberi) [reply locale/ar/download/current.md]: هذه ليست ترجمتي لكن لاباس .. ان فقط قمت بتعديل المثل من ناحية التنسيق ليكون مماثل للملف الأصلي ..

File: locale/ar/download/releases.md [hunk 2]
```diff
@@ -2,7 +2,6 @@
 layout: download-releases.hbs
 title: النسخ سابقة
 modules: "يشير <code>NODE_MODULE_VERSION</code> إلى رقم نسخة النود جي اس الخاص بالواجهة الثنائية (ABI) للتطبيق، وهو يستعمل لتحديد أي من نسخ النود جي اس المنتجة كملفات ثنائية خاصة بالسي ++ يمكنها ان تحمل بدون الحاجة إلى إعادة انتاجها. في وقت سابق، كان يتم تخزينها على شكل قيم ستة عشرية (hex) اما الآن فهي تمثل على شكل ارقام صحيحة"
```
Comment (Aissaoui-Ahmed) [inline locale/ar/download/releases.md]: هنا أيضا

File: locale/ar/download/index.md [hunk 3]
```diff
@@ -22,8 +22,7 @@ downloads:
 additional:
     headline: منصات إضافية
     intro: >
-        يقوم أعضاء من مجتمع النود جي اس بالاشراف على نسخ غير رسمية من النود جي اس لمنصات أخرى.
-        يجب التذكير أن هذه النسخ غير مدعومة من الفريق الأساسي للنود جي اس و قد لا تكون على نفس مستوى التطوير الخاص بالاصدارات الحالية للنود جي اس.
+        يقوم أعضاء من مجتمع النود جي اس بالاشراف على نسخ غير رسمية من النود جي اس لمنصات أخرى. يجب التذكير أن هذه النسخ غير مدعومة من الفريق الأساسي للنود جي اس و قد لا تكون على نفس مستوى التطوير الخاص بالاصدارات الحالية للنود جي اس.
```
Comment (Aissaoui-Ahmed) [inline locale/ar/download/index.md]: هنا أيضا

**Review by Aissaoui-Ahmed — CHANGES_REQUESTED** [review]

**Comment by Aissaoui-Ahmed** [comment]
thank you for your contribution @3imed-jaberi

**Review by celyes — CHANGES_REQUESTED** [review]
looks perfect, except whenever there is the word 'Node.js', leave it as it is in the English writing... this is how the guide recommends as well as what is found across all other translations.

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Comment by Aissaoui-Ahmed** [comment]
LGTM :100:

**Comment by 3imed-jaberi** [comment]
@celyes , please check this  ..

**Review by celyes — APPROVED** [review]
Absolutely beautiful !

**Pull request merged**